parm/config/config.fcst

```diff
@@ -186,7 +191,14 @@ if [[ "$CDUMP" == "gdas" ]] ; then # GDAS cycle specific parameters
     export restart_interval=${restart_interval:-6}
 
     # For IAU, write restarts at beginning of window also
-    if [ $DOIAU = "YES" ]; then export restart_interval="6 9"; fi
+    if [[ "$SDATE" = "$CDATE" ]]; then
+       if [ $DOIAU = "YES" ]; then export restart_interval="3 6"; fi
+    else
+       if [ $DOIAU = "YES" ]; then export restart_interval="6 9"; fi
+    fi
```

Comment:
Would it make more sense to change this block to 

if [ $DOIAU = "YES" ]; then 
      export restart_interval="6 9"
      if [[ "$SDATE" = "$CDATE" ]]; then  export restart_interval="3 6"; fi
fi

Reply:
Yes that is better, I will adjust config.fcst and commit the change to feature/wave2global. Will also adjust config.efcs.